### ISSUE TITLE
Elastic Search

### ISSUE THREAD

**Issue by kkimberley** [kkimberley/taroko_school#10]
Scenario: Path search
As a user
When I visit paths#index page
Then I should be able to search context I interest by keyword

@GDD is it suitable to use Elastic Search for searching?

**Comment by GGD** [comment]
I think it's overkill at this stage, let's use pure SQL query for now.

**Comment by kkimberley** [comment]
ok, thanks!
one more question, if there are many data in paths#index, 
how can I page the list, is it ok for use gem kaminari? I saw it at 
Ruby on Rails 實戰聖經.

**Comment by GGD** [comment]
Yeah if we contain logs of data in one page, it's nice to use kaminari.
However, that function is "nice to have" instead of "must have", let's focus on "must have".
